### PR TITLE
Update to Golang 1.9.2

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by ruflin** [comment]
jenkins, package it

File: dev-tools/packer/docker/xgo-image-deb6/go-1.8.3/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ MAINTAINER Tudor Golubenco <tudor@elastic.co>
 
 # Configure the root Go distribution and bootstrap based on it
 RUN \
-  export ROOT_DIST="https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz" && \
+  export ROOT_DIST="https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz" && \
   export ROOT_DIST_SHA1="838c415896ef5ecd395dfabde5e7e6f8ac943c8e" && \
```
Comment (andrewkroh) [inline dev-tools/packer/docker/xgo-image-deb6/go-1.8.3/Dockerfile]: Pretty sure the SHA1 sums need updated too.

Comment (ruflin) [reply dev-tools/packer/docker/xgo-image-deb6/go-1.8.3/Dockerfile]: good point. updated both. Also renamed the directory which is called `1.8.3`.

**Comment by ruflin** [comment]
jenkins, package it

**Comment by ruflin** [comment]
Testing packaging locally ...

**Comment by ruflin** [comment]
Run the packaging locally and it seems to work as expected.

**Comment by ruflin** [comment]
jenkins, test it

**Comment by ruflin** [comment]
jenkins, retest it

**Comment by ruflin** [comment]
Build currently fails with
```
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-QaRIki/PyYAML/
../libbeat/scripts/Makefile:209: recipe for target 'python-env' failed
```

**Comment by ruflin** [comment]
jenkins, retest it

**Pull request merged**